runbotics/runbotics-desktop/src/core/websocket/listeners/process.listener.ts

```diff
@@ -31,12 +32,32 @@ export class ProcessListener {
         private readonly websocketService: WebsocketService
     ) {}
 
+    private async startKeepingConnection() {
+        const intervalId = setInterval(() => {
+            const message: Message = {
+                event: BotWsMessage.KEEP_ALIVE,
+                payload: null
+            };
+
+            this.websocketService.emitMessage(message, true);
+        }, 60000);
+
+        this.keepAliveStatus = { intervalId, isActive: true };
+    }
+
+    private async stopKeepingConnection() {
```

Comment:
I dont like this naming `start/stop keeping connection`
I suggest `beginKeepAliveInterval` and `clearKeepAliveInterval`

Reply:
additionally, what async about those methods?